index.html

```diff
@@ -761,6 +818,16 @@ <h2>Dictionary <a class="idlType">DisplayMediaStreamConstraints</a>
                   the audio track. If <code>false</code>, the {{MediaStream}}
                   will not contain an audio track.</p>
                 </dd>
+                <dt>
+                  <dfn><code>currentTabPreference</code></dfn> of type {{CurrentTabPreferenceEnum}},
+                  defaulting to {{CurrentTabPreferenceEnum/none}}
+                </dt>
+                <dd>
+                  Signals application preference for whether the current
+                  [=browser=] [=display surface=] should be included in the list
+                  of [=display surfaces=] offered to the user.
+                  The user agent MAY regard this hint.
```

Comment:
"Current" is not defined. Also reverse the MAY to clarify this prose is optional:
```suggestion
                  If present, signals application preference for whether the
                  [=browser=] [=display surface=] associated with
                  [=this=]'s [=relevant global object=]'s [=associated Document=]'s
                  [=top-level browsing context=] should be among the choices
                  offered to the user. The user agent MAY ignore this hint.
```